### PR TITLE
fix(gatsby-admin): Setup Gatsby Admin site

### PR DESCRIPTION
**Pull request by mxstbr**
☂️ This is an umbrella PR that smaller PRs get added to! ☂️

This is a first Gatsby site for Gatsby Admin, based on querying the Recipes GraphQL server. PRs that were reviewed & merged into this umbrella PR:

- [x] Add mutations to the recipes GraphQL server (#23335)
- [x] Use mutations of recipes GraphQL server in Gatsby Admin (#23337)

### CODE REVIEW

**Comment by mxstbr** [comment]
> "Stoiber" is a typo. Did you mean "Stolider"?

No TypoCI 🤣

**Comment by laurieontech** [comment]
Big thing for me here is a barebones README on how to use/run it. I'd love to test it out and currently don't know how lol

**Comment by mxstbr** [comment]
The `unit_tests_www` test failures are unrelated flakes. I can't view the "Cloud Tests" failures (no permission), but they are likely unrelated as well.

This is ready to be reviewed and merged!

File: packages/gatsby-admin/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Gatsby Admin
+
+A visual interface to configure your Gatsby site.
+
+We have not packaged this nicely yet, so its not installable
```
Comment (laurieontech) [inline packages/gatsby-admin/README.md:5]:
```suggestion
We have not packaged this nicely yet, so it is not installable.
```

**Comment by laurieontech** [comment]
Ok, pushed the README changes based on our walkthrough @mxstbr.
I think the best way forward is to make the umbrella PR and merge this and the mutations change into that?

File: packages/gatsby-admin/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+We have not packaged this nicely yet, so it is not installable.
+
+## How to develop it
+
+However, you can do some manual set up in order to work with it locally. Follow these steps:
+
+1. Navigate to the monorepo and open the `packages/gatsby-admin` directory.
+2. In that directory, run `yarn develop`.
+   > If you see eslint errors you'll need to temporarly replace all references to `___loader` with `window.___loader`.
```
Comment (laurieontech) [inline packages/gatsby-admin/README.md:13]:
```suggestion
   > If you see eslint errors you'll need to temporarly replace all references to `___loader` with `window.___loader` inside of `gatsby-link/index.js`.
```

**Comment by mxstbr** [comment]
This'll just be the umbrella PR, as it's the base for all of them anyway!

Since all the individual changes (i.e. the other PRs) have been approved, I'm going to get their CI green and merge them into here, and then get this first version into `master` 👍

**Comment by mxstbr** [comment]
This is now the Umbrella PR for Gatsby Admin and is ready to go!

 I merged both child PRs into here once they were reviewed and approved (#23335 and #23337), and this was also reviewed and approved so it is ready to be merged into `master`?! 👍

File: packages/gatsby-admin/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+We have not packaged this nicely yet, so it is not installable.
+
+## How to develop it
+
+However, you can do some manual set up in order to work with it locally. Follow these steps:
+
+1. Navigate to the monorepo and open the `packages/gatsby-admin` directory.
+2. In that directory, run `yarn develop`.
+   > If you see eslint errors you'll need to temporarly replace all references to `___loader` with `window.___loader` inside of `gatsby-link/index.js`.
```
Comment (jxnblk) [inline packages/gatsby-admin/README.md:13]:
```suggestion
   > If you see eslint errors you'll need to temporarily replace all references to `___loader` with `window.___loader` inside of `gatsby-link/index.js`.
```

File: packages/gatsby-admin/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Gatsby Admin
+
+A visual interface to configure your Gatsby site.
+
+We have not packaged this nicely yet, so it is not installable.
```
Comment (LekoArts) [inline packages/gatsby-admin/README.md:5]: Your post on Slack indicated that it's now possible. So is this info outdated or that feature in another PR/branch?

File: packages/gatsby-admin/src/urql-client.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
+const {
+  createClient,
+  // useMutation,
```
Comment (LekoArts) [inline packages/gatsby-admin/src/urql-client.ts:3]: Some unused imports here

File: packages/gatsby-admin/README.md (hunk 4 above)
Comment (mxstbr) [reply packages/gatsby-admin/README.md:5]: In another PR (#23368), but @freiksenet pointed out that we don't actually want a plugin as we have to hook into the parent process—so I have to finish #22600 (which requires finishing #22759, which is what I'm working on right now) first 😬

File: packages/gatsby-admin/src/pages/index.tsx [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+        id: $name
+      }) {
+        id
+        name
+      }
+    }
+  `)
+
+  return (
+    <button
```
Comment (LekoArts) [inline packages/gatsby-admin/src/pages/index.tsx:69]: Let's add a `aria-label="Delete"` or
```js
aria-label={`Delete ${name}`}
``` 
here as explained in Example 1: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

**Pull request merged**